src/test/java/org/sonar/plugins/stash/StashRequestFacadeTest.java

```diff
@@ -368,6 +368,7 @@ public void testGetStashPullRequestIdThrowsException() throws StashConfiguration
 
   @Test
   public void testPostCommentPerIssue() throws Exception {
+    when(config.getIssueSeverityThreshold()).thenReturn(Severity.INFO);
```

Comment:
Can you move all these calls to the setUp() method?

Reply:
moved calls to setUp()